media/js/base/mozilla-fxa-link.js

```diff
@@ -43,12 +86,21 @@ if (typeof window.Mozilla === 'undefined') {
             return;
         }
 
+        var userVer = parseFloat(client._getFirefoxVersion());
+        var useUITourForFxA = userVer > 78 && Mozilla.UITour;
         var fxaSigninLink = document.querySelectorAll('.js-fxa-cta-link');
 
         for (var i = 0; i < fxaSigninLink.length; i++) {
             var link = fxaSigninLink[i];
             var mozillaOnlineLink = link.getAttribute('data-mozillaonline-link');
 
+            if (useUITourForFxA) {
+                // intercept the flow and submit the form using the UITour API instead.
+                // In the future we should fully migrate to this API for Firefox Desktop login.
+                link.addEventListener('auxclick', this.interceptFxANavigation);
```

Comment:
When I try to right click one of these links it immediately opens a new tab, as opposed to letting me use the context menu (for example to inspect the element or the link URL). This feels a little unexpected/disorienting - is there another way to deal with someone trying to trigger one of these links in a new tab?

Reply:
Hm I think the only way would be to make it a `<button>`?

Reply:
It might be easier to add `role=button` to the links for now, and then disable the context menu?

```javascript
link.setAttribute('role', 'button');
link.oncontextmenu = function(e) {
    e.preventDefault();
};
```

This then avoids the odd menu behaviour, and should at least provide user agents with a hint as to the role of the element.

Reply:
Awesome, thank you for the tip! Yesterday I tried replacing the DOM node with the button. It worked well for the most part except the styling. I think your suggestion should work better